LambdaS3FileZipper/LambdaS3FileZipper.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Lets not make this change.  We should keep our libraries at `netstandard2.0` as it maximizes compatibility across different consumers.

Reply:
Good point @brianfeucht! Done via https://github.com/litmus/aws-lambda-s3-zipper-dotnet/pull/37/commits/6efe6f53619995527f500acbd0e94b8a60b4e437! ✔️